system/Helpers/form_helper.php

```diff
@@ -713,13 +713,13 @@ function form_close(string $extra = ''): string
 	 * Grabs a value from the POST array for the specified field so you can
 	 * re-populate an input field or textarea
 	 *
-	 * @param string  $field      Field name
-	 * @param string  $default    Default value
-	 * @param boolean $htmlEscape Whether to escape HTML special characters or not
+	 * @param string       $field      Field name
+	 * @param string|array $default    Default value
+	 * @param boolean      $htmlEscape Whether to escape HTML special characters or not
 	 *
 	 * @return string
 	 */
-	function set_value(string $field, string $default = '', bool $htmlEscape = true): string
+	function set_value(string $field, $default = '', bool $htmlEscape = true)
```

Comment:
Why remove the `: string` return typehint?

Reply:
Becasue the method return value can be string array. Having type hint in such case will throw error.

I see that I missed to update the return type in phpdoc. It should be string|array

Reply:
That is what I'm thinking earlier. Good to see the changes.